src/Core/Models/Api/Request/Accounts/RegisterRequestModel.cs

```diff
@@ -26,6 +26,8 @@ public class RegisterRequestModel : IValidatableObject, ICaptchaProtectedModel
     public Guid? OrganizationUserId { get; set; }
     public KdfType? Kdf { get; set; }
     public int? KdfIterations { get; set; }
+    public int? KdfMemory { get; set; }
```

Comment:
Same. Should there be some validation?

Reply:
Added validation:

https://github.com/bitwarden/server/blob/29af2b3ef7693b42c9cf307c5ef8f146b8d42d3e/src/Api/Models/Request/Accounts/KdfRequestModel.cs#L27-L41